VMEncryption/main/EncryptionSettingsUtil.py

```diff
@@ -210,8 +211,9 @@ def controller_id_and_lun_to_settings_data(scsi_controller, lun_number):
     def write_settings_file(self, data):
         """ Dump encryption settings data to JSON formatted file on key volume """
         self.increment_index()
-        with open(self.get_settings_file_path(), 'w', 0) as outfile:
-            json.dump(data, outfile)
+        output = json.dumps(data)
```

Comment:
add comment clarifying ascii encoding to preserve backwards compatibility (not adding UTF8 BOM)

Reply:
added